packages/inline-edit/src/utils/getStyles.ts

```diff
@@ -0,0 +1,105 @@
+import {
+  css,
+  CSSObject,
+  FlattenSimpleInterpolation,
+  FlattenInterpolation,
+} from 'styled-components';
+import get from 'lodash.get';
+import { mergeDeep } from 'utilitify';
+import memoize from 'micro-memoize';
+
+import {
+  getThemedState,
+  getAppearanceTheme,
+  ITheme,
+  AllType,
+} from '@xcritical/theme';
+
+import { inlineEditThemeNamespace, defaultInlineEditTheme } from '../theme';
+import {
+  IInlineEditTheme,
+  ICommonProps,
+  IReturnFunction,
+  GetPropStyles,
+} from '../interfaces';
+
+
+export const inlineEditTheme = memoize((
+  theme: ITheme<IInlineEditTheme>,
+  propertyPath?: string | string[],
+): AllType => {
+  const func = getThemedState(inlineEditThemeNamespace, defaultInlineEditTheme);
+  return func(theme, propertyPath);
+});
+
+const inlineEditAppearanceTheme = memoize((
+  theme: ITheme<IInlineEditTheme> = {},
+  appearanceName: string,
+  baseAppearance: string,
+  propertyPath?: string | string[],
+): ITheme<IInlineEditTheme> | any => {
+  const func = getAppearanceTheme(inlineEditThemeNamespace, defaultInlineEditTheme);
+  return func(theme, appearanceName, propertyPath, baseAppearance);
+});
+
+export const getInlineEditThemeStylesByProperty = memoize((
+  theme: ITheme<IInlineEditTheme>,
+) => (propertyPath: string[]): CSSObject => {
+  const customBlanketTheme = get(theme, inlineEditThemeNamespace);
+  const mergedTheme = mergeDeep(defaultInlineEditTheme, customBlanketTheme);
+
+  return get(mergedTheme, propertyPath);
+});
+
+export const getBaseStyle = memoize(({
```

Comment:
Is it unnecessary function.
Inside `ReadViewContentWrapper` you can just `${({ theme }) => inlineEditTheme(theme)}`.